prefs/src/main/java/io/github/fate_grand_automata/prefs/Preferences.kt

```diff
@@ -82,6 +82,10 @@ class PreferencesImpl @Inject constructor(
 
     override val screenshotDropsUnmodified by prefs.screenshotDropsUnmodified
 
+    override val screenshotBond by prefs.screenshotBond
+
+    override var hidePlayButtonForScreenshot by prefs.hidePlayButtonForScreenshot
```

Comment:
Preference can't be changed by users right now

Reply:
Sorry for the late reply, only read your comments now. I was talking about how `hidePlayButtonForScreenshot` can't be changed.

Reply:
I would prefer if the variable had `@ScriptScope` or `@ServiceScoped`, but I think so far preferences are the only thing shared by both which can affect the UI.